README.md

```diff
@@ -162,19 +176,16 @@ Once the new js file has been added, any new projects created will use the updat
         }
       }
     }
-    </pre>
-
-3. You should probably add a `<platform>:{}` entry to the `Gruntfile` compile arrays.
-4. Make sure your native implementation executes the following JavaScript once all of the native side is initialized and ready: `require('cordova/channel').onNativeReady.fire()`.
-5. The `deviceready` event is important. To make sure that the stock
-   common JavaScript fires this event off, the device and network
-   connection plugins must successfully be instantiated and return
-   information about the connectivity and device information. The
-   success callbacks for these plugins should include calls to
-   `require('cordova/channel').onCordovaInfoReady.fire()` (for device
-   information) and
-   `require('cordova/channel').OnCordovaConnectionReady.fire()` (for
-   network information).
-6. Last but certainly not least: add yourself to the contributors list!
-   It's in the `package.json` file in the root of this repository. You
-   deserve it!
+    ```
+
+4. Make sure your native implementation executes the following JavaScript once all of the native side is initialized and ready:
+
+    ```javascript
+    require('cordova/channel').onNativeReady.fire()
+    ```
+
+5. The `deviceready` event is important. To make sure that the stock common JavaScript fires this event off, the device and network connection plugins must successfully be instantiated and return information about the connectivity and device information. The success callbacks for these plugins should include calls to `require('cordova/channel').onCordovaInfoReady.fire()` (for device information) and `require('cordova/channel').OnCordovaConnectionReady.fire()` (for network information).
```

Comment:
```suggestion
```

AFAICT this information is outdated. These channels do not exist in `cordova-js`. To delay the `deviceready` event until some other channel has fired, `waitForInitialization` is used now. This function is currently used only by plugins, not by platforms. Which makes sense, because platforms already have the `onNativeReady` channel which `deviceready` waits for anyway. So all in all we can chuck that point.